Formula/unit.rb

```diff
@@ -0,0 +1,60 @@
+class Unit < Formula
```

Comment:
I think it would be better to name is `NginxUnit`

Reply:
Yes, `unit` is a common word, but at the moment there is no package with this name, we already build rpm and deb packages with this name and it is good to have consistent naming.
Brew already have `ant` and `go` which is not prefixed.

@vbart , how do you like proposed name: `NginxUnit`?

Reply:
Our [official packages](https://unit.nginx.org/installation/#official-packages) for various Linux distribution have name "unit". [FreeBSD port](https://www.freshports.org/www/unit) uses name "unit", some [3rd-party Linux repositories](https://pkgs.alpinelinux.org/packages?name=unit*) also prefer to call package as "unit".

I think it will be less confusing for users if the package name in Homebrew will be similar.

Reply:
Since we have a bunch of modules, maybe we need to create and maintain our own Tap?